flit/__init__.py

```diff
@@ -58,11 +62,14 @@ def main(argv=None):
     elif args.subcmd == 'install':
         from .install import Installer
         Installer(args.ini_file, user=args.user, symlink=args.symlink).install()
+    elif args.subcmd == 'register':
+        from .wheel import WheelBuilder
+        WheelBuilder(args.ini_file).register()
     elif args.subcmd == 'init':
         from .init import TerminalIniter
         TerminalIniter().initialise()
     else:
-        sys.exit('No command specified')
+        sys.exit('No command specified, use --help for more informations')
```

Comment:
I wonder if argparse has a method to print the help info - rather than telling people to do `--help`, we could just act as if they had. No need for that to be in this PR, just a thought.